src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.c

```diff
@@ -1,40 +1,93 @@
 /*
  * File: z_eff_ss_extra.c
  * Overlay: ovl_Effect_Ss_Extra
- * Description:
+ * Description: Horseback Archery Points
  */
 
 #include "z_eff_ss_extra.h"
 
 typedef enum {
-    /* 0x00 */ SS_EXTRA_0,
-    /* 0x01 */ SS_EXTRA_1,
-    /* 0x02 */ SS_EXTRA_2,
-    /* 0x03 */ SS_EXTRA_3,
-    /* 0x04 */ SS_EXTRA_4,
-    /* 0x05 */ SS_EXTRA_5,
-    /* 0x06 */ SS_EXTRA_6,
-    /* 0x07 */ SS_EXTRA_7,
-    /* 0x08 */ SS_EXTRA_8,
-    /* 0x09 */ SS_EXTRA_9,
-    /* 0x0A */ SS_EXTRA_A,
-    /* 0x0B */ SS_EXTRA_B,
-    /* 0x0C */ SS_EXTRA_C,
+    /* 0x00 */ SS_EXTRA_OBJ_IDX,
+    /* 0x01 */ SS_EXTRA_RISING_TIMER,
+    /* 0x02 */ SS_EXTRA_SCORE_IDX,
+    /* 0x03 */ SS_EXTRA_SCALE
 } EffectSsExtraRegs;
 
 u32 EffectSsExtra_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsExtra_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsExtra_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-/*
+static s16 sScores[] = { 30, 60, 100 };
+
 EffectSsInit Effect_Ss_Extra_InitVars = {
     EFFECT_SS_EXTRA,
     EffectSsExtra_Init,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/effects/ovl_Effect_Ss_Extra/EffectSsExtra_Init.s")
+static UNK_PTR D_809A4070[] = { 0x06000000, 0x06000480, 0x06000900 };
+
+extern Gfx D_06000DC0[];
+
+u32 EffectSsExtra_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx) {
+    EffectSsExtraInitParams* initParams = (EffectSsExtraInitParams*)initParamsx;
+    ObjectContext* objCtx = &globalCtx->objectCtx;
+    s32 objIdx;
+    u32 temp;
+
+    objIdx = Object_GetIndex(objCtx, OBJECT_YABUSAME_POINT);
+
```

Comment:
we currently call this objBankIndex in most places. we can probably shorten it to Idx eventually, but for now we can keep it consistent with the rest